tools/interactive/interactivetool_napari.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="interactive_tool_napari" tool_type="interactive" name="napari" version="@VERSION@+galaxy0" profile="23.1">
+	<description>cell image analysis software</description>
+    <macros>
+        <token name="@VERSION@">0.4.18</token>
+    </macros>
+    <requirements>
+	    <container type="docker">registry.git.embl.de/grp-cbbcs/abcdesktop-apps/napari-galaxy:latest</container>
```

Comment:
Can you please version this container? Ideally same version as the tool?
Is it ok for you to push this container to quay.io or dockerhub?